scene/gui/code_edit.cpp

```diff
@@ -2691,11 +2691,16 @@ void CodeEdit::_filter_code_completion_candidates() {
 			option.icon = completion_options[i].get("icon");
 			option.default_value = completion_options[i].get("default_value");
 
-			max_width = MAX(max_width, font->get_string_size(option.display).width);
+			int offset = 0;
+			if (option.default_value.get_type() == Variant::COLOR) {
+				offset = get_line_height();
```

Comment:
`get_line_height` is quite expense to run at the moment, would recommend calculating this outside the loop. Same for below.

Reply:
Thanks, fixed